arm/avhclient/aws_backend.py

```diff
@@ -820,21 +861,31 @@ def send_remote_command(self, command_list, working_dir, fail_if_unsuccess=True)
         This is a mandatory AVH backend method.
         """
         self._init()
-        logging.info(f"avh: command_list = {command_list}")
+        logging.debug("aws:command_list = %s", command_list)
         response = self.send_ssm_shell_command(
             command_list=command_list,
             working_dir=working_dir
         )
 
+        log_init = '='*80
+        logging.info(log_init) if enable_logging_info else logging.debug(log_init)
+
         for i in response.keys():
-            logging.info(f"avh:{i} = {response[i].strip()}")
+            info_desc = f"aws:send_remote_command:{i} = {response[i].strip()}"
+            logging.info(info_desc) if enable_logging_info else logging.debug(info_desc)
```

Comment:
```suggestion
            logging.log(logging.INFO if enable_logging_info else logging.DEBUG, "aws:send_remote_command:%s = %s", i, response[i].strip())
```

Reply:
It looks we change globally with that. I just want to for a particular command do that. It should then respect what was initially selected.

Reply:
Never mind, it does not do that!! I will make the change.